plugins/typescript/src/core/schemaToTypeAliasDeclaration.ts

```diff
@@ -544,6 +544,27 @@ const getJSDocComment = (
       }, schema)
     : schema;
 
+  const getJsDocIdentifier = (value: unknown) => {
+    const multilineEndChar = "*/";
+
+    if (typeof value === "string" && !value.includes(multilineEndChar)) {
+      return f.createIdentifier(value);
+    }
+
+    if (
+      typeof value === "object" &&
+      !JSON.stringify(value).includes(multilineEndChar)
+    ) {
+      return f.createIdentifier(JSON.stringify(value));
+    }
+
+    if (typeof value === "boolean" || typeof value === "number") {
+      return f.createIdentifier(value.toString());
+    }
+
+    return f.createIdentifier("[see original specs]");
```

Comment:
What do you think of adding a comment like this for folks that may see this placeholder in the future?
```suggestion
    // Value is not stringifiable
    // See https://github.com/fabien0102/openapi-codegen/issues/36, https://github.com/fabien0102/openapi-codegen/issues/57
    return f.createIdentifier("[see original specs]");
```